File: Assignment__18/Question2.java
//Write a program to print all even numbers up to N.

import java.util.Scanner;

class function
{
    void DisplayEven(int iNo)
    {
        int iCnt = 0;

        if(iNo < 0)
        {
            iNo = -iNo;
        }

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            if(iCnt % 2 == 0)
            {
                System.out.print(iCnt + " ");
            }
        }
    }
}

class Question2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        int iValue = sobj.nextInt();

        function fobj = new function();
        
        fobj.DisplayEven(iValue);

    }
}

File: Assignment__41/Question2.c
#include<stdio.h>

int SumDigits(int iNo)
{
    if(iNo == 0) 
    {
        return 0;
    }

    return (iNo % 10) + SumDigits(iNo / 10);
}

int main()
{
    int iValue = 0;
    int iResult = 0;

    printf("Enter the number : ");
    scanf("%d", &iValue);

    iResult = SumDigits(iValue);

    printf("Sum of digits is : %d", iResult);

    return 0;
}
